src/content/reference/cluster-upgrades/_index.md

```diff
@@ -97,15 +115,15 @@ In particular this means:
 - As a consequence of draining, Pods running on a node will be rescheduled to other nodes.
 - Once the master node is taken down and recreated, the Kubernetes API will be unavailable for a short time.
 
-**Note**: By default, tenant clusters have one master node each. [Consider using a High Availability masters setup](#checklist-ha-masters) to avoid API downtime during upgrades.
+**Note**: By default, workload clusters have one master node each. [Consider using a High Availability masters setup](#checklist-ha-masters) to avoid API downtime during upgrades.
 
 ### Provider-specific details for AWS
```

Comment:
Why not call the provider-specific session just <provider name> specific details?

Reply:
Good point